tools/cldr-apps/src/main/java/org/unicode/cldr/web/STFactory.java

```diff
@@ -908,6 +691,11 @@ public VoteResolver<String> getResolver(String path) {
             return getResolver(peekXpathData(path), path, null);
         }
 
+        @Override
+        public VoteResolver<String> getResolver(String path, VoteResolver<String> r) {
+            return getResolver(peekXpathData(path), path, r);
```

Comment:
minor quibble: I prefer to put caller before callee -- easier to read with higher-level methods higher in the file